### DISCUSSION TITLE
Update to include self added fields

### DISCUSSION THREAD

**Discussion by anon533566** [Royal-Modding/FS19_PrecisionFarmingAddon#7] [Ideas]
this mod is great and i been waiting for something like this, however at the moment it only "half" works on my end, it detects the "original" field sizes, but however does not scan the fields later on added and or the fields that i connected together creating bigger then original size fields.  see map for what i refer too. if text does not make much sense sorry :) other then that this mod is a great addon.  
![afbeelding](https://user-images.githubusercontent.com/82620684/114952138-e8cba100-9e55-11eb-8b3b-69ab969994f4.png)

**Comment by TyKonKet** [comment]
I understand your point of view, unfortunately for now farming simulator works with farmlands and field definitions, both are defined by mappers and the seconds cannot overflow the firsts.
If you want to make it work, you have to open the map in GE, make a single farmland and extend the field definition.
Otherwise if you can wait some weeks, we are working on a possible solution, but we are not sure that it will work.

**Reply by anon533566** [reply]
hey guys wanted to check in see if there was any luck so far or progress in finding/working on a solution. just curious. no pressure.

**Reply by TyKonKet** [reply]
Hey, we haven't been able to make it work, I'm so sorry.
We are players too and it's frustrating even for us when you extend a field and it can't be analyzed, but we can't don nothing about that a the moment.

**Comment by anon533566** [comment]
I understand thats why i posted the idea its not critism as it works as intended originally and i’m glad i have much less to scan now more time to play alone and focus on other less time consuming tasks as there is plenty of others haha.

I’m happy to wait if you need a tester i’m happy to test too.
